returnn/torch/engine.py

```diff
@@ -8,6 +8,10 @@
 
 import os
 import torch
+import time
+from torch.distributed import init_process_group, destroy_process_group
+from torch.nn.parallel import DistributedDataParallel as DDP
+from torch.distributed.algorithms.join import Join
```

Comment:
Do not import such generic names (`Join`) into the main namespace.

Reply:
Do you even need this here? I thought this is now in `returnn.torch.distributed`?